### PR TITLE
[INLONG-1911] add pulling metric of prometheus, pulling metric value will not be resetted.

### LINKED ISSUE #1911: [Bug] Some questions about the metric implementation in the common module
### What happened

I analyzed the metric implementation in the common module[1], and there are a few issues that need to be corrected, or they can’t be connected with the external standard JMX call.
a. To be implemented based on MXBean instead of MBean;
    MBean can only provide the call of the basic class, and the composite class needs MXBean to define
b. The name format of the constructed Beanobject should be unified:
    To be constructed based on ObjectName("XXXX:type=YYYYY") format, where XXXX and YYYYYare arbitrary, but ":type=" is a fixed format
c. Cannot only have get-and-reset() API:
    If we only have the get-and-reset() API, when we look at Counter type indicators, we still need the caller to keep the previous data and accumulate it by itself, which is unreasonable.

1. https://github.com/apache/incubator-inlong/tree/master/inlong-common/src/main/java/org/apache/inlong/commons/config/metrics

### What you expected to happen

Use standard JMX tools, for example, jconsole, for interface access

### How to reproduce

Use standard JMX tools, for example, jconsole, for interface access

### Environment

_No response_

### InLong version

master

### InLong Component

Other for not specified component

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by luchunliang**
[INLONG-1911] add pulling metric of Prometheus, pulling metric value will not be reset.

### Title Name: [INLONG-1911][DataProxy]  add pulling metric of Prometheus, pulling metric value will not be reset.

Fixes #1911

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduces a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by aloyszhang** [comment]
@baomingyu @EMsnap PTAL

**Review by gosonzhang — APPROVED** [review]
+1

**Review by dockerzhang — APPROVED** [review]

File: inlong-dataproxy/dataproxy-source/pom.xml [hunk 1]
```diff
@@ -46,19 +46,13 @@
 		<dependency>
 			<groupId>org.powermock</groupId>
 			<artifactId>powermock-module-junit4</artifactId>
-			<version>2.0.2</version>
+			<version>${powermock.version}</version>
```
Comment (inter12) [inline inlong-dataproxy/dataproxy-source/pom.xml:49]: The third-party dependent version needs to be external to the parent pom  <dependencymanagement> file

Comment (inter12) [reply inlong-dataproxy/dataproxy-source/pom.xml:49]: dependencymanagement

Comment (luchunliang) [reply inlong-dataproxy/dataproxy-source/pom.xml:49]: ok, move dependency to pom.xml of inlong-dataproxy

**Pull request merged**